crates/symbolicator-service/src/services/objects/data_cache.rs

```diff
@@ -238,7 +235,7 @@ impl CacheItemRequest for FetchFileDataRequest {
             object,
 
             scope: self.0.scope.clone(),
-            cache_key: self.get_cache_key(),
+            cache_key: CacheKey::from_scoped_file(&self.0.scope, &self.0.file_source),
```

Comment:
where do we still need this?

Reply:
Only for tracing logs.

Reply:
fair enough. we should eventually clean those up at some point